src/shout_subgroup/models.py

```diff
@@ -43,6 +43,7 @@ class SubgroupModel(Base):
     created_at = Column(DateTime, default=datetime.now(timezone.utc))
     updated_at = Column(DateTime)
     users = relationship("UserModel", secondary=users_subgroups_join_table, backref="subgroups")
+    table_args = (UniqueConstraint('group_chat_id', 'name', name='_group_chat_id_name_uc'))
```

Comment:
Focus: Added this table constraint. group chat id and (subgroup)name together has to be unique for each row